src/main/java/com/launchdarkly/client/Variation.java

```diff
@@ -81,22 +81,22 @@ public boolean matchTarget(LDUser user) {
   static class TargetRule {
     String attribute;
     String operator;
-    List<String> values;
+    List<Object> values;
```

Comment:
This looks like a bug-- this wouldn't have worked with non-string target rule values